cognite/extractorutils/configtools.py

```diff
@@ -163,6 +163,7 @@ class CogniteConfig:
     project: str
     api_key: Optional[str]
     idp_authentication: Optional[AuthenticatorConfig]
+    data_set_id: Optional[int]
```

Comment:
For the dotnet utils, we decided to leave` data-set-id` and `external-id-prefix` out of the lib because we were not enforcing these on time series creation. That is, the client applications set those properties. Not sure if it is the same for python

Reply:
Yeah, I'm not really sold on it being there for all extractors either, but for the `BaseConfig` class to make sense, I believe this is the best approach for now.